sdk/textanalytics/ai-text-analytics/src/generated/index.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
```

Comment:
Make sure to either change these copyright headers or to skip the generated folder from eslint! (I'm not sure if you've considered this already)

Reply:
`eslint` does not lint code in the `generated` directory. What is wrong with these copyright headers?